server.R

```diff
@@ -429,9 +435,14 @@ shinyServer(function(input, output, session) {
       # associates metadata with data and returns manifest id
       manifest_id <- synapse_driver$associateMetadataWithFiles(
         synStore_obj,
-        "./tmp/synapse_storage_manifest.csv", folder_synID()
+        "./tmp/synapse_storage_manifest.csv",
+        folder_synID(),
+        manifest_record_type = template_type
```

Comment:
@rrchai manfiest_record_type can be "table", "entity" or "both". The past default behavior of schematic was equivalent to setting manfiest_record_type to "entity". We can transition to setting manifest_record_type to "both".

For references, here is the relevant [docstring]( https://github.com/Sage-Bionetworks/schematic/blob/a9301999d407b10caa7370478ae96cff32f32472/schematic/store/synapse.py#L606)

Reply:
@milen-sage oops. Thanks for reminding. I was definitely misunderstood the "manifest_record_type". 
Since I've already merged the changes to the `main`, I will make a hotfix pr on `main` and create a new tag (v22.04.2).